portal/config/eproms/site_persistence_file.json

```diff
@@ -21,7 +21,6 @@
         "CUSTOM_PATIENT_DETAIL = True\n",
         "LOCALIZED_AFFILIATE_ORG = 'TrueNTH Global Registry'\n",
         "ORGS_W_CUSTOM_INVITES = ['TrueNTH Global Registry', 'IRONMAN']\n",
-        "MEDIDATA_RAVE_FIELDS = ['deceased', 'study_id', 'consent_status', 'dob', 'org']\n",
```

Comment:
Well, you're not going to like my $.02, but to me, this variable name is undesirable.
1. it's defining which fields to hide/protect
2. it's vendor specific
If it defines a list of buttons/controls/fields to disable or hide - lets call it something like "PROTECTED_FIELDS" and the matching "PROTECTED_ORG"

But not critical, so I'm just submitting as a comment.

Reply:
>lets call it something like "PROTECTED_FIELDS" and the matching "PROTECTED_ORG"

I like @pbugni s suggestion, and since we have to formalize these configs (add to `config.py`) anyway, can you rename them to something more generic?